.golangci.yml

```diff
@@ -46,4 +46,9 @@ issues:
     - path: _test\.go
       linters:
         - errcheck
-        - gosec
\ No newline at end of file
+        - gosec
+
+    # FIXME temporarily suppress this.
```

Comment:
deprecate bearer strategy followed by code logic removal and alias token strategy in it for backward compatibility.